firebase-segmentation/firebase-segmentation.gradle

```diff
@@ -57,7 +57,7 @@ android {
     compileSdkVersion project.targetSdkVersion
 
     defaultConfig {
-        minSdkVersion project.minSdkVersion
+        minSdkVersion 21
```

Comment:
All firebase SDKs require at most API level 16, while some 14. Was it a conscious choice to require 21?

Reply:
While switch to use SQLiteOpenHelper, we don't need to choose this directory any more.